File: year2/mpp/java/MotorcycleClientServer/MotorcycleCommon/src/main/java/domain/Race.java
package domain;

public class Race extends Entity<Integer> {
    private Integer capacity;

    public Race() {
        capacity = null;
    }

    public Race(Integer capacity) {
        this.capacity = capacity;
    }

    public Integer getCapacity() {
        return capacity;
    }

    public void setCapacity(Integer capacity) {
        this.capacity = capacity;
    }

    @Override
    public String toString() {
        return "Race " + getId() + ", capacity " + capacity + "; ";
    }
}

File: year2/mpp/c#/MotorcycleGrpcServer/MotorcycleCommon/Domain/Race.cs
using System;

namespace MotorcycleContest.Domain.Entities
{
    [Serializable]
    public class Race : Entity<int>
    {
        public int Capacity { get; set; }

        public Race()
        {
            this.Capacity = -1;
        }

        public Race(int capacity)
        {
            this.Capacity = capacity;
        }

        public override string ToString()
        {
            return "Race " + ID + ", capacity " + Capacity + "; ";
        }
    }
}
